docker-compose/demo-app.yaml

```diff
@@ -25,4 +25,14 @@ services:
   webapp:
     build: ../src/webapp
     ports:
-      - 50000:8080
\ No newline at end of file
+      - 50000:8080
+
+  guest_webapp:
```

Comment:
To add seperation of concerns, don't hard-code your services here. Keep those details inside your service, and simply reference your services' files. Something like this:
```
include:
  - ../src/guest_webapp/docker-compose.yaml     # uses port 8081
  - ../src/staff_webapp/docker-compose.yaml     # uses port 8082
services:
  nginx:
    image: nginx:stable-alpine-slim
    ports:
      - 8089:80
      - 80:80
    volumes:
      - ./configs/nginx.conf:/etc/nginx/nginx.conf

  accounts:
    build: ../src/accounts
    ports:
      - 50001:8080
      
  permissions:
    build: ../src/permissions
    ports:
      - 50002:8080
      
  sessions:
    build: ../src/sessions
    ports:
      - 50003:8080

  webapp:
    build: ../src/webapp
    ports:
      - 50000:8080

```